hadoop-tools/hadoop-distcp/src/test/java/org/apache/hadoop/tools/util/TestDistCpUtils.java

```diff
@@ -1248,28 +1245,24 @@ public void testCompareFileLengthsAndChecksums() throws IOException {
         replFactor, srcSeed);
     DFSTestUtil.createFile(fs, dstWithChecksum1, 1024,
         replFactor, srcSeed);
-    DistCpUtils.compareFileLengthsAndChecksums(fs, srcWithChecksum1,
-        null, fs, dstWithChecksum1, false);
-    DistCpUtils.compareFileLengthsAndChecksums(fs, srcWithChecksum1,
+    DistCpUtils.compareFileLengthsAndChecksums(1024, fs, srcWithChecksum1,
```

Comment:
howe about we factor out all these 1024s into a single variable, including the one on L1246